### PR TITLE
Update MaterialDrawer

### PR DESCRIPTION
**Pull request by AnuthaDev**
Updated MaterialDrawer Library to 7.0.0

### CODE REVIEW

File: example/src/main/java/com/github/appintro/example/ui/MainActivity.kt [hunk 1]
```diff
@@ -34,17 +37,19 @@ class MainActivity : AppCompatActivity() {
                                 .withIdentifier(2)
                                 .withName("Permissions")
                 )
-                .withOnDrawerItemClickListener {_, _, drawerItem ->
-                    var fragment: Fragment = MainTabsFragment()
+                .withOnDrawerItemClickListener(object: Drawer.OnDrawerItemClickListener{
+                    override fun onItemClick(view: View?, position: Int, drawerItem: IDrawerItem<*>): Boolean {
+                        var fragment: Fragment = MainTabsFragment()
```
Comment (cortinico) [inline example/src/main/java/com/github/appintro/example/ui/MainActivity.kt]: Please convert this to a lambda as before:

```
withOnDrawerItemClickListener { _, _, drawerItem ->
                    var fragment: Fragment = MainTabsFragment()

                    when (drawerItem.identifier.toInt()) {
                        1 -> fragment = MainTabsFragment()
                        2 -> fragment = PermissionTabsFragment()
                    }

                    supportFragmentManager.beginTransaction().replace(R.id.frame_container, fragment).commit()
                    false
                }

```

**Comment by cortinico** [comment]
> By the way, it looked better before in Kotlin 😆

Yeah should definitely be fixed.

**Comment by AnuthaDev** [comment]
@cortinico Fixed?

**Comment by cortinico** [comment]
@AnuthaDev can you please address this comment? https://github.com/AppIntro/AppIntro/pull/749/files#r374397026

**Comment by AnuthaDev** [comment]
Will do tomorrow for sure

**Comment by AnuthaDev** [comment]
Hey @cortinico Sorry for not responding, I am busy with exams, and won't be able to  contribute anytime soon, so it would be great if you could take over this PR or close this and create another...😢

**Comment by cortinico** [comment]
Closing as we removed Material Draware in #816

**Pull request closed without merging**